docs/release-notes.txt

```diff
@@ -3,6 +3,134 @@
 Release Notes
 =============
 
+Version 0.13
+------------
+
+Version 0.13.0
+^^^^^^^^^^^^^^
+
+**Release Date:** August 20, 2020
+
+**New Features**
+
+- Allow experiments to connect to the master using TLS.
+
+- Allow agent's TLS verification to skip verification or use a custom certificate for the master.
+
+- For ``TFKerasTrial`` and ``EstimatorTrial``, add functionality to ``context.wrap_dataset()`` which allows users to skip sharding their training dataset for distributed training. If using this option, users are responsible for splitting their dataset in such a manner that every GPU (rank) sees unique data.
```

Comment:
A bit more detail on how you might use this "functionality" would be nice -- e.g., is it activated by passing a particular argument to wrap_dataset?

Also, let's link to the API docs for wrap_dataset.